app.py

```
@@ -261,6 +380,36 @@ def generar_id_pedido():
 def generar_codigo_recompensa():
     return "".join(random.choices(string.ascii_uppercase + string.digits, k=6))
 
+def check_pending_rewards(user_emoji):
+    user = User.query.get(user_emoji)
+    if not user: return []
```

Ruff (0.12.2)

Multiple statements on one line (colon)

(E701)